### PR TITLE
Added new user route

### PR DESCRIPTION
**Pull request by BenVuong**
I added a new route for new users who are not logged in. I am currently controlling this with the variable isLoggedIn in App.js. Pending authentication and roles creation: this would help frontend developers to use this variable to create a new route locally.

### CODE REVIEW

**Review by S78901 — DISMISSED** [review]
This looks pretty good Ben. Thanks for formatting these files with Prettier.

**Review by MisheelSodgerel — APPROVED** [review]

File: web-ui/src/pages/BackgroundInformationPage.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  text-transform: unset !important;
+
+  position: absolute;
+  left: 5%;
+  bottom: 5%;
+}
+
+#introductionSurvey {
+  text-transform: unset !important;
+
```
Comment (S78901) [inline web-ui/src/pages/BackgroundInformationPage.css:11]: Avoid extra line breaks

File: web-ui/src/pages/AccessCodePage.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import react from "react";
```
Comment (S78901) [inline web-ui/src/pages/AccessCodePage.jsx:1]: This should be:
`import React from "react";`

The lowercase is making it fail the gradle build.

File: web-ui/src/components/routes/NewUserRoutes.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import React, { useContext } from "react";
```
Comment (S78901) [inline web-ui/src/components/routes/NewUserRoutes.jsx:1]: Delete the unused 'useContext' so the gradle build does not fail.

`import React from "react";`

**Review by li-brandon — APPROVED** [review]

**Pull request merged**